src/graphql/get-users.js

```diff
@@ -65,3 +65,43 @@ export const GET_USER_WATCHLIST = gql`
     }
   }
 `;
+
+export const GET_USERS_WATCHLIST = gql`
+  query GetUsersWatchlist {
```

Comment:
This query should get a list of recommended movies based off the _**current user's**_ preferences rather than the current user's watchlist and watched slops. The wishlist and watched are a part of the me goblin profile page. The page and logic for user preference currently does not exist.